controls/package_spec.rb

```diff
@@ -65,3 +65,38 @@
     it { should_not be_installed }
   end
 end
+
+## can also be syslog-ng...
+control 'package-07' do
```

Comment:
I am not sure why we should enforce a specific logging environment. Syslog is only one way to achieve logging